src/main/io/serial_cli.c

```diff
@@ -154,7 +154,7 @@ static void cliVersion(char *cmdline);
 static void cliRxRange(char *cmdline);
 static void cliPFlags(char *cmdline);
 
-#ifndef SKIP_TASK_STATISTICS
+#if !defined(SKIP_TASK_STATISTICS) && !defined(SKIP_CLI_RESOURCES)
```

Comment:
I didn't want to disable all the statistics, since they are quite useful for development, so this just disables the resources.